docs/builtin-libraries.rst

```diff
@@ -0,0 +1,272 @@
+Built-in Libraries
```

Comment:
Nice, but harder to maintain that a board vs library supported. I would assume if a library is supported by a board, some (if not all) examples will be supported.

Reply:
Section has been taken from Wiki and will be adapted incrementally